### ISSUE TITLE
How to configure Postgres settings

### ISSUE THREAD

**Issue by PeterJCLaw** [CircleCI-Public/cimg-postgres#47]
Since switching to cimg/postgres:11.13 (from circleci/postgres:11-alpine-ram) we've seen intermittent errors like:
```
django.db.utils.OperationalError: out of shared memory
HINT:  You might need to increase max_locks_per_transaction.
```

While I'm not sure if the issue is actually related to the change of image, I can't see in the docs any means to configure the Postgres instance. As this setting is one which requires a postgres restart, ideally it's something I'd configure before the container starts.

Is this a case of creating a custom image, or is there a way to overlay Postgres' configuration via CI config?

**Comment by felicianotech** [comment]
reviewing

**Comment by ChrisLusted** [comment]
also looking how I can configure the Postgres instance. did you find a fix @PeterJCLaw?

**Comment by PeterJCLaw** [comment]
I don't think I did.

**Comment by ChrisLusted** [comment]
shame, thanks for the reply

**Comment by JalexChen** [comment]
Closed with #67

**Comment by dheerajck** [comment]
Can we add these parameters to config 
https://discuss.circleci.com/t/can-postgres-config-postgresql-conf-file-be-changed-using-circle-yml/10621/4
```
fsync=off
synchronous_commit=off
full_page_writes=off
bgwriter_lru_maxpages=0
```
